sdk/storage/Azure.Storage.Files.DataLake/src/Sas/DataLakeSasQueryParameters.cs

```diff
@@ -13,7 +13,7 @@ namespace Azure.Storage.Sas
     /// parameters.  You can construct a new instance using
     /// <see cref="DataLakeSasBuilder"/>.
     ///
-    /// For more information, <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/constructing-a-service-sas"/>.
+    /// For more information, <see href="https://docs.microsoft.com/rest/api/storageservices/create-service-sas">Create a service SAS</see>.
```

Comment:
I'm seeing enough URLs that have changed that I'm concerned. Are we just 404'ing any customers who have linked or bookmarked older URLs, or are they all redirecting appropriately? This one does redirect, as have a couple others I've tried. As long as that happens, great.